extensions/audio/src/test/java/ai/djl/audio/FFmpegAudioFactoryTest.java

```diff
@@ -37,20 +37,22 @@ public void setUp() throws IOException {
 
     @Test
     public void testFactory() throws IOException {
-        Audio audio = AudioFactory.getInstance().fromFile(Paths.get("build/test/test_01.wav"));
+        Audio audio = new FFmpegAudioFactory(null).fromFile(Paths.get("build/test/test_01.wav"));
```

Comment:
Rather than call `new FFmpegAudioFactory(null)`, it may be cleaner to just add a constructor with the default configuration